lms/djangoapps/edraak_marketing_email/views.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth.models import User
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from edraak_marketing_email.helpers import (
+    subscribe_to_marketing_emails,
+    unsubscribe_from_marketing_emails
+)
+
+
+class UnsubscribeUserAPIView(APIView):
+    """
+    Unsubscribe user from marketing emails
+    """
+    authentication_classes = []
```

Comment:
Just to confirm, this is meant to work without authentication right?

Reply:
Right.